### PR TITLE
chore: added analytics and speedinsights from vercel

### PR DESCRIPTION
**Pull request by 0xtiti**
## Checklist before requesting a review

- [x] I have conducted a self-review of my code.
- [ ] If it is a core feature, I have included comprehensive tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one sentence about this update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,8 @@
     "@mui/material-nextjs": "5.15.6",
     "@rainbow-me/rainbowkit": "2.0.1",
     "@tanstack/react-query": "5.20.5",
+    "@vercel/analytics": "^1.2.2",
+    "@vercel/speed-insights": "^1.0.10",
```
Comment (0xArdy) [inline package.json:39]: remember to lock the dependencies version

File: yarn.lock
Comment (0xArdy) [inline yarn.lock:1]: this file isn't necessary

**Review by 0xArdy — CHANGES_REQUESTED** [review]

**Comment by 0xArdy** [comment]
you can add `closes <SUP XX>` in the pr description and thenit automatically mark the issue as done once the pr is merged

File: package.json (hunk 1 above)
Comment (0xtiti) [reply package.json:39]: sorry about that!

Comment (0xtiti) [reply package.json]: https://github.com/defi-wonderland/superchain-bridge-ui/commit/a8000c604958f24c8d6caacc2891766bb39497c7

File: yarn.lock
Comment (0xtiti) [reply yarn.lock:1]: https://github.com/defi-wonderland/superchain-bridge-ui/commit/a8000c604958f24c8d6caacc2891766bb39497c7

**Review by 0xArdy — APPROVED** [review]
lgtm! :rocket:

**Comment by turtlemoji** [comment]
reviewed

**Pull request merged**